.github/workflows/tests.yml

```diff
@@ -41,6 +41,10 @@ jobs:
     - uses: ./.github/actions/setup-test-environment
     - name: Install JavaScript dependencies with Yarn
       run: yarn check || yarn install --frozen-lockfile;
+    - name: Run webpacker setup
+      run: rails g webpack:install && rails g webpack:install:react && rails g react:install
```

Comment:
Hmm, @noi5e this caused:

```
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:90:in `connect': Access denied for user 'runner'@'localhost' (using password: NO) (Mysql2::Error::ConnectionError)
```

That's a bit odd, what's going on there?

Reply:
Hmm, that error does seem really weird. I'm not sure, it seems unrelated to React and webpacker? Going to run the tests again and see if it persists.

Here's the very start of that stacktrace:

```
from /home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/railties-5.2.3/lib/rails/commands.rb:18:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

Doesn't it seem like something went wrong at a very fundamental level with `rails` and `railsties`? Could be a CI problem.

Reply:
The tests are running again now, this appears in the stacktrace, so it seems like it's going to end similarly as the first run:
<img width="1090" alt="Screen Shot 2021-03-02 at 4 47 23 PM" src="https://user-images.githubusercontent.com/4361605/109735160-265ebe00-7b77-11eb-9550-2a537fc6ed9b.png">

However, looking further "upstream" it looks like the problem is caused by yarn not installing dependencies correctly? I wonder why?
<img width="1099" alt="Screen Shot 2021-03-02 at 4 46 49 PM" src="https://user-images.githubusercontent.com/4361605/109735134-1c3cbf80-7b77-11eb-82a6-a498bd35decb.png">

Reply:
Hmm, yarn shouldn't relate to mysql though... i wonder. Let's see; certainly earlier commands related to the database ought to be using the correct mysql info, so maybe we adapt to how those worked...